src/video/wayland/SDL_waylandvideo.c

```diff
@@ -570,7 +570,8 @@ static void display_handle_done(void *data,
         native_mode.w = driverdata->native_width;
         native_mode.h = driverdata->native_height;
     }
-    native_mode.refresh_rate = (int)SDL_round(driverdata->refresh / 1000.0); /* mHz to Hz */
+    native_mode.refresh_rate_numerator = (int)SDL_round(driverdata->refresh / 1000.0);
```

Comment:
This should be:
```
native_mode.refresh_rate_numerator = driverdata->refresh;
native_mode.refresh_rate_denominator = 1000;
```
Or maybe we need to look at how refresh is calculated and save even higher precision?

Reply:
this was already done in a previous commit ( you commented an old commit)!

refresh comes from wayland listener, so we cannot have higher precision ..
(display_handle_mode, ,in wl_output_listener output_listener 
https://wayland-client-d.dpldocs.info/wayland.client.protocol.wl_output_listener.mode.html